pulp_smash/tests/ostree/api_v2/test_copy.py

```diff
@@ -0,0 +1,79 @@
+# coding=utf-8
+"""Test copy of specific content from/to OSTree repositories."""
+import unittest
+from urllib.parse import urljoin
+
+from pulp_smash import api, config, utils
+from pulp_smash.constants import OSTREE_BRANCH, OSTREE_FEED, REPOSITORY_PATH,\
+    OSTREE_FEED_SMALL
```

Comment:
In almost all cases, line continuations with backslashes should be avoided. Use parentheses:

```python
from pulp_smash.constants import (
...
)
```

One reason that backslashes are avoided is that comments can't be placed after them:

```
from pulp_smash.constants import FOO, \  # my comment
    BAR
```

The compiler reads this as:

```
from pulp_smash.constants import FOO, \  # my comment BAR
```